pkg/metadata/host/host_test.go

```diff
@@ -183,3 +183,10 @@ func TestGetProxyMeta(t *testing.T) {
 	meta = getProxyMeta()
 	assert.Equal(t, meta.ProxyBehaviorChanged, true)
 }
+
+func TestGetNetworkInfo(t *testing.T) {
+	ni, err := GetNetworkInfo()
+	require.NoError(t, err)
+	require.Greater(t, ni.EphemeralPortStart, uint16(1024))
```

Comment:
Is it possible for this to be flaky, like if it runs on a machine with a different set of defaults?

Reply:
I suppose it's possible, but the ephemeral port range "shouldn't" be set lower than 1024. 
I'm open to other suggestions how to test.

Reply:
`> 1024` should be pretty safe. `<= 1024` are "well-known ports" https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers#Well-known_ports

Reply:
I think Hasans suggestion is a good one.  

I've seen the range start at either 1024 or 32768 depending on the Linux distro so the above test should cover it.

Reply:
This test is already doing > 1024. Perhaps it should be doing >= 1024, since the dynamic range starts at 1024 according to wikipedia.